src/lvl_script_value.c

```diff
@@ -944,7 +944,7 @@ void script_process_value(unsigned long var_index, unsigned long plr_range_id, l
     {
         if (plr_range_id == ALL_PLAYERS)
         {
-            for (i = PLAYER3; i >= PLAYER0; i--)
+            for (i = ALL_PLAYERS-1; i >= PLAYER0; i--)
```

Comment:
I don't really see why it goes backwards, 
`for (i = 0; i < PLAYERS_COUNT; i++)`
sounds more like normal code here

Reply:
ok